chaco/scales/formatters.py

```diff
@@ -236,7 +239,7 @@ class IntegerFormatter(BasicFormatter):
     def format(self, ticks, numlabels=None, char_width=None, fill_ratio=0.3):
         """ Formats integer tick labels.
         """
-        return map(str, map(int, ticks))
+        return list(map(str, map(int, ticks)))
```

Comment:
I would suggest to use `itertools.imap` so that python 2.7 does not get a performance penalty for creating and destroying multiple list objects